rust/worker/src/blockstore/arrow/provider.rs

```diff
@@ -243,7 +243,7 @@ impl BlockManager {
                                     return None;
                                 }
                             };
-                            tracing::info!("Read {:?} bytes from s3", buf.len());
+                            tracing::info!("Read {:?} bytes from s3 for block get", buf.len());
```

Comment:
as a nit, these additions are not that useful IMO and make the trace longer as this is already wrapped in an outer span. I think its ok to not basically shove outer spans names into inner events but instead rely on the association.